build_tooling/parallel_test.sh

```diff
@@ -33,7 +33,8 @@ function worker() {
     MSYS=winsymlinks:nativestrict ln -s "$(realpath "$tooling_dir/../python/tests")" $new_root/
     cd $new_root
 
-    $catch python -m pytest -v --show-capture=no --log-file="$TEST_OUTPUT_DIR/pytest-logger.$group.log" \
+    export ARCTICDB_RAND_SEED=$RANDOM
+    $catch python -m pytest -v -n $PYTEST_THREADS --show-capture=no --log-file="$TEST_OUTPUT_DIR/pytest-logger.$group.log" \
```

Comment:
DO NOT use pytest-xdist (`-n`) with pytest-splits (`--splits`)!!

Reply:
Why not? The tests seems stable for the most part and this is addressing the problem that:
- we cannot run the test only with xdist (due to mongo) and splits seems to remedy this somewhat 
- xdist is helping speed up the integration tests on windows by bringing the time down from ~100 min to ~20 min

Reply:
Running two test splitting/parallelisation plugins is certainly an unsupported scenario, and I wouldn't even be confident that it's correctly running all the tests and reporting all the test outcomes.

Also, the effect of the two plugins will multiply, and one might end up running way more tests in parallel by mistake.

For the mongo issue, see my other comment.

Reply:
According to the logs, it is behaving as expected.
It first splits the tests and then parallelizes only over the selected tests.

Reply:
If they appear to work, then I am okay with this setup.

After my fixtures PRs is merged, `pytest-split` can be removed.

Reply:
I've just "unresolved" this thread so it's easy to find later